ajax/libs/shadydom/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "shadydom",
+  "filename": "shadydom.min.js",
+  "version": "1.0.8",
+  "description": "Shadow DOM polyfill",
+  "keywords": [
+    "shady-dom",
+    "shadydom",
+    "shadow-dom",
+    "shadowdom",
+    "web-components",
+    "webcomponents",
+    "polyfill",
+    "shim"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/webcomponents/shadydom.git"
+  },
+  "author": "The Polymer Authors",
+  "homepage": "https://webcomponents.org/polyfills",
```

Comment:
Though this follows the upstream, and "Shadow DOM polyfill" is introduced in the link, the content of this page seems for another GitHub repo https://github.com/WebComponents/webcomponentsjs.

Reply:
@extend1994 I did not understand, is there need to change something?

Reply:
@sashberd No for now. But I didn't sure about it. That's why I request review from Peter.